src/transport/UTransport.cpp

```diff
@@ -10,3 +10,61 @@
 // SPDX-License-Identifier: Apache-2.0
 
 #include "up-cpp/transport/UTransport.h"
+#include "up-cpp/datamodel/validator/UUri.h"
+#include "up-cpp/datamodel/validator/UMessage.h"
+
+using namespace uprotocol;
+using namespace uprotocol::transport;
+namespace UriValidator = uprotocol::datamodel::validator::uri;
+namespace MessageValidator = uprotocol::datamodel::validator::message;
+
```

Comment:
The implementations for `UTransport` should be in the `uprotocol::transport` namespace. `using namespace` is not adequate here.

```suggestion
namespace uprotocol::transport {

namespace UriValidator = datamodel::validator::uri;
namespace MessageValidator = datamodel::validator::message;

```

Being inside the namespace also gives the same benefits of the `using namespace` statements - the current and parent namespaces are automatically available.